Ports/rvvm/patches/disable-threaded-io.patch

```diff
@@ -0,0 +1,45 @@
+commit f897929a4616f0b0a88bab36c7fc2a9c5bbb3365
+Author: LekKit <50500857+LekKit@users.noreply.github.com>
+Date:   Fri Mar 22 10:05:07 2024 +0200
+
+    Disable threaded IO on Serenity
+
```

Comment:
This does not follow the proper patch format. Please move the patch and readme somewhere safe, and use `./package.sh dev` to create a commit with the desired changes, title, and description (which will then be automatically exported to the correct patch format on exit).